Ghidra/Framework/Project/src/main/java/ghidra/framework/plugintool/dialog/PluginManagerComponent.java

```diff
@@ -127,7 +127,7 @@ void manageAllPlugins() {
 			descriptionlabel.setForeground(Color.GRAY);
 			descriptionlabel.setBorder(BorderFactory.createEmptyBorder(5, 0, 0, 0));
 			descriptionlabel.setVerticalAlignment(SwingConstants.TOP);
-			descriptionlabel.setPreferredSize(new Dimension(300, 60));
+			descriptionlabel.setPreferredSize(new Dimension(300, 80));
```

Comment:
That's just a hack, it will overflow again if the text is changed to something longer.